src/components/ModalWindow/styles.ts

```diff
@@ -0,0 +1,79 @@
+import { StyleSheet } from 'react-native';
+
+export const styles = StyleSheet.create({
+  modalContainer: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  modelWindow: {
+    flex: 1,
+    marginTop: '30%',
+    margin: 0,
+    padding: 12,
+
+    justifyContent: 'space-between',
+    alignItems: 'center',
+
+    borderTopLeftRadius: 40,
+    borderTopRightRadius: 40,
+    backgroundColor: '#fff',
+  },
+  separator: {
+    position: 'absolute',
+    top: 0,
+
+    backgroundColor: '#fff',
+    borderRadius: 4,
+
+    marginTop: -12,
+
+    width: '25%',
+    height: 4,
+  },
+  productContainer: {
+    alignItems: 'center',
+    justifyContent: 'center',
+
+    marginHorizontal: 50,
+    marginVertical: 25,
+  },
+  productImage: {
+    width: 250,
+    height: 250,
+    borderRadius: 10,
+  },
+  storoLogo: {
+    position: 'absolute',
+    left: 7,
+    top: 7,
+
+    zIndex: 1,
+  },
+  nameText: {
+    fontFamily: 'Roboto-Medium',
```

Comment:
Since it can be repeated many times it's better to move it as a constant in `styles` folder (`styles/fonts`)